modules/plugins/lsp/config.nix

```diff
@@ -116,7 +116,7 @@ in {
         end
 
         local capabilities = vim.lsp.protocol.make_client_capabilities()
-        ${optionalString usingNvimCmp "capabilities = require('cmp_nvim_lsp').default_capabilities()"}
+        -- ${optionalString usingNvimCmp "capabilities = require('cmp_nvim_lsp').default_capabilities()"}
```

Comment:
Why is this commented out?

Reply:
me being dumb

this little maneuver is preventing me from lazy loading cmp so I commented it out to test things

will add it back in + lazy load lspconfig

Reply:
I've decided to skip on lazy-loading lspconfig since it would pretty much mean rewriting the entire language module. So cmp is now not lazy loaded